wallet/src/de/schildbach/wallet/ui/dashpay/InvitationFragmentViewModel.kt

```diff
@@ -87,21 +106,18 @@ open class InvitationCreatedFragmentViewModel(application: Application) : BasePr
         }
     }
 
-    private val pubkeyHash = walletApplication.wallet.currentAuthenticationKey(AuthenticationKeyChain.KeyChainType.INVITATION_FUNDING).pubKeyHash
-    val inviteId = Address.fromPubKeyHash(walletApplication.wallet.params, pubkeyHash).toBase58()
-
     val identityIdLiveData = MutableLiveData<String>()
 
     val invitationLiveData = Transformations.switchMap(identityIdLiveData) {
-        liveData (Dispatchers.IO) {
+        liveData(Dispatchers.IO) {
             emit(invitationDao.loadByUserId(it)!!)
         }
     }
 
-    val invitation : Invitation
+    val invitation: Invitation
```

Comment:
For me there is a crash here with the `get()` method.  Not sure why this didn't crash when QA was testing.  It could be related to other changes I had in my libraries, so lets have another look at this after my PR's are merged and we can fix it at that point.